src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1473,46 +1473,52 @@ void SolverInterfaceImpl::computePartitions()
   }
 }
 
-void SolverInterfaceImpl::mapWrittenData()
+void SolverInterfaceImpl::computeMappings(utils::ptr_vector<MappingContext> contexts, std::string mappingType)
 {
-  PRECICE_TRACE();
   using namespace mapping;
   MappingConfiguration::Timing timing;
-  // Compute mappings
-  for (impl::MappingContext &context : _accessor->writeMappingContexts()) {
-    timing         = context.timing;
-    bool rightTime = timing == MappingConfiguration::ON_ADVANCE;
-    rightTime |= timing == MappingConfiguration::INITIAL;
+  for (impl::MappingContext &context : contexts) {
+    timing      = context.timing;
+    bool mapNow = timing == mapping::MappingConfiguration::ON_ADVANCE;
+    mapNow |= timing == mapping::MappingConfiguration::INITIAL;
```

Comment:
Since you already import the namespace functions above you could drop the namespace prefix here. If you still want to keep it explicit like this then I would propose to remove the `namespace` above.

Reply:
Fixed in d275cf93ebe63626e45d7394e975e34690cbec46.